src/mount/fuse/mount_config.cc

```diff
@@ -92,7 +92,7 @@ struct fuse_opt gSfsOptsStage2[] = {
 	SFS_OPT("bandwidthoveruse=%lf", bandwidthoveruse, 1),
 	SFS_OPT("sfsdirentrycachesize=%u", direntrycachesize, 0),
 	SFS_OPT("nostdmountoptions", nostdmountoptions, 1),
-	SFS_OPT("sfsignoreflush", ignoreflush, 1),
+	SFS_OPT("sfsignoreflush=%d", ignoreflush, 1),
```

Comment:
Consider 0 as the last parameter or better the default value.

Reply:
Done